docs/Project.toml

```diff
@@ -1,4 +1,6 @@
 [deps]
+ClimaComms = "3a4d1b5c-c61d-41fd-a00a-5873ba7a1b0d"
+ClimaCommsMPI = "5f86816e-8b66-43b2-912e-75384f99de49"
```

Comment:
Do we need this for the docs?

Reply:
It looks like one of the examples/tutorials is using allreduce!

Reply:
I think it was just using an older version of the dependency. If you just `Pkg.update()` it will pick up the correct one. These don't need to be here as they're not called directly.